esmvaltool/config-developer.yml

```diff
@@ -19,10 +19,19 @@
 ---
 
 CMIP6:
-  input_dir: 'CMIP/[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[version]'
+  input_dir:
+    default: '/'
+    DKRZ: 'CMIP/[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
```

Comment:
Maybe the initial `CMIP` could be user defined in config-user.yml? Or do you think it is better like this?